.github/dependabot.yml

```diff
@@ -5,6 +5,14 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+    group:
```

Comment:
Isn't it `groups` instead of `group`? I've not used this. Just looking at [the docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups).

Reply:
:facepalm: